webapp/src/components/gallery/gallery.tsx

```diff
@@ -25,6 +31,8 @@ type Props = {
 
 const Gallery = (props: Props): JSX.Element => {
     const {activeView, board, cards} = props
+    const limited = useAppSelector(getHiddenCard)
```

Comment:
nit - `limited` sounds like a `boolean` variable. Why not name this `hiddenCards`?